hphp/runtime/base/runtime-option.h

```diff
@@ -617,6 +617,8 @@ struct RuntimeOption {
    * kept low to avoid the risk of collecting a sample while we're      \
    * processing a previous sample. */                                   \
   F(uint32_t, PerfMemEventSampleFreq, 80)                               \
+  /* PPC64 Option: Minimum immediate size to use TOC */                 \
+  F(uint16_t, PPC64MinTOCImmSize, 64)                                   \
```

Comment:
Can you move this above the `Profiling flags` section divider?  This is not a profiling flag.  (You can also section of a new set of flags for PPC64, if you expect you'll have more.)